src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/RetargetingTests.cs

```diff
@@ -589,6 +589,26 @@ public interface Test : short { }
             Assert.Null(retargetingType.BaseType());
         }
 
+        [Fact]
+        [WorkItem(1141012, "https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1141012")]
+        public void RetargetFixedSizeField()
+        {
+            var source = @"
+public unsafe struct Test
+{
+    public fixed byte buffer[12];
+}
+";
+
+            var comp = CreateCompilation(source);
+            var sourceAssembly = (SourceAssemblySymbol)comp.Assembly;
+            var retargetingAssembly = new RetargetingAssemblySymbol(sourceAssembly, isLinked: false);
+            retargetingAssembly.SetCorLibrary(MissingCorLibrarySymbol.Instance); // Need to do this explicitly since our retargeting assembly wasn't constructed using the real mechanism.
+            var retargetingType = retargetingAssembly.GlobalNamespace.GetMember<NamedTypeSymbol>("Test");
+            var retargetingField = retargetingType.GetMember<RetargetingFieldSymbol>("buffer");
+            Assert.True(retargetingField.IsFixedSizeBuffer);
+        }
+
```

Comment:
Thanks for covering the change. However, this type of test for retargeting symbols has fallen in disfavor, I was told.

Here's a template for creating retargeting symbols: create two compilations with two different corlibs. Such test can be put in `FixedSizeBufferTests`.

```
            var source = @"
namespace System
{
    public struct ValueTuple<T1, T2>
    {
        public T1 Item1;
        public T2 Item2;
        public string field;
    }
}";
            var comp = CreateCompilation(source, targetFramework: TargetFramework.Mscorlib40, assemblyName: "customValueTuple");
            comp.VerifyDiagnostics();
           
            var comp3 = CreateCompilation("", references: new[] { comp.ToMetadataReference() }, targetFramework: TargetFramework.Mscorlib46);
            var retargetingValueTupleType = (NamedTypeSymbol)comp3.GlobalNamespace.GetMember<NamespaceSymbol>("System").GetMembers("ValueTuple").Single();
            Assert.IsType<RetargetingNamedTypeSymbol>(retargetingValueTupleType);
```